receiver/otelarrowreceiver/factory.go

```diff
@@ -19,9 +19,9 @@ import (
 const (
 	defaultGRPCEndpoint = "0.0.0.0:4317"
 
-	defaultMemoryLimitMiB    = 128
-	defaultAdmissionLimitMiB = defaultMemoryLimitMiB / 2
-	defaultWaiterLimit       = 1000
+	defaultMemoryLimitMiB  = 128
+	defaultRequestLimitMiB = defaultMemoryLimitMiB / 2
```

Comment:
```suggestion
	defaultRequestLimitMiB = 128
```

Reply:
This is a subjective feeling, but I'd like to not make the impression that these two are tied or that admission should be half of the memory limit. It's harder to change defaults later, so I'd prefer to raise this now.

Reply:
Fixed